osf/models/mixins.py

```diff
@@ -2270,7 +2270,10 @@ def copy_editable_fields(self, resource, auth=None, alternative_resource=None, i
         but the alternative_resource will be a Node.  DraftRegistration fields will trump Node fields.
         TODO, add optional logging parameter
         """
-        self.set_editable_attribute('title', resource, alternative_resource)
+        from osf.models.draft_node import DraftNode
+        if not isinstance(resource, DraftNode):  # Force the user to add their own title.test_create_draft_registration_without_node
+            self.set_editable_attribute('title', resource, alternative_resource)
```

Comment:
This is breaking the abstraction hierarchy. I'm also not sure if it's necessary. Could/should be able to add an optional param to control this the way we did for contributors. Maybe even accept a list of `excluded_attributes`?

`for attribute in ['title', 'description', 'category', 'node_license']:
    if attribute not in excluded_attributes:
        self.set_editable_attribute(attribute, resource, alternative_resource)
`

Then, when calling this from DraftRegistration.create_from_node you can exclude the title in the No Project case. Followup would be to use that same mechanism for contributors.